dashboard/test/models/pd/application/facilitator1819_application_test.rb

```diff
@@ -6,6 +6,13 @@ class Facilitator1819ApplicationTest < ActiveSupport::TestCase
     self.use_transactional_test_case = true
     setup_all do
       @regional_partner = create :regional_partner
+      @workshop = create :pd_workshop
+      @application = create :pd_facilitator1819_application
+      @application_with_fit_workshop = create :pd_facilitator1819_application, fit_workshop_id: @workshop.id
+    end
+    setup do
+      @application.reload
+      @application_with_fit_workshop.reload
```

Comment:
Most of this is just test optimization. We were creating applications in many test cases. Here create it once and reuse.